tests/test_application/test_application.py

```diff
@@ -78,6 +80,26 @@ def test_application_contents_with_none_contents(flask_test_client):
     assert response.status_code == 400
 
 
+def test_application_map_contents_response(app_context):
+    """
+    GIVEN: our service running with app_context fixture.
+    WHEN: we post expected data to Application class to map contents.
```

Comment:
🤔 Where does the actual post request take place?

Reply:
its not a post method. Supposed to be send 😄